modules/gdscript/gdscript_editor.cpp

```diff
@@ -1264,7 +1265,19 @@ static void _find_identifiers_in_base(const GDScriptCompletionIdentifier &p_base
 				}
 				return;
 			} break;
-			case GDScriptParser::DataType::ENUM:
+			case GDScriptParser::DataType::ENUM: {
+				String type_str = base_type.native_type;
+				StringName type = type_str.substr(0, type_str.find("."));
```

Comment:
```suggestion
				StringName type = type_str.get_slicec('.', 0);
```
Better method for this specific use